tests/python/test_python_frontend.py

```diff
@@ -3,25 +3,21 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Owner(s): ["module: nvfuser"]
 
-from copy import deepcopy
 from functools import partial
 import itertools
 import math
 import re
 import random
-from typing import List, Callable
-import unittest
+from typing import List
 
 import torch
 import torch.nn.functional as F
-from torch.testing._internal.common_utils import run_tests, TEST_WITH_ROCM, TestCase
-from torch.testing._internal.jit_utils import RUN_CUDA
 import torch._refs as refs
 import torch._prims as prims
 
 from nvfuser import (
-    FusionCache,
     FusionDefinition,
+    FusionCache,
```

Comment:
Why changing the order? That being said, it may make more sense to sort alphabetically. (If you use Vim, there's a thing called select and `:sort`.)

Reply:
This was unintentional, possibly crept in while moving things around.